app/segments/drill/[id]/submission/input.js

```diff
@@ -459,6 +606,18 @@ export default function Input({ drillInfo, setToggleResult, setOutputData }) {
                 Error! All input fields must be filled!
               </Banner>
 
+              <Banner
```

Comment:
Can we move away from Banners? I feel like the buttons and textboxes shifting down is really annoying.

Reply:
Banners have been changed to Dialogs.